src/main/java/org/generationcp/breeding/manager/customcomponent/SaveListAsDialog.java

```diff
@@ -246,16 +246,13 @@ public GermplasmList getGermplasmListToSave() {
 		if (selectedList != null && !SaveListAsDialog.FOLDER_TYPE.equalsIgnoreCase(selectedList.getType())) {
 			this.germplasmList = this.getSelectedListOnTree();
 
-			// Needed for overwriting
-			this.source.setCurrentlySavedGermplasmList(this.germplasmList);
```

Comment:
Have you tested the case where a another list will be overwritten by current list, @gelli-leafnode ?

Reply:
Hi @darla-leafnode, 
For the scenario you mentioned above, I believe this is when generating cross where list can be edited - Upon confirming to QA this is no longer applicable.
I also tested the scenario: Existing list is open and changes in list builder will be saved: Output the existing list from List Manager will be closed.